Dockerfile

```diff
@@ -18,6 +18,12 @@ RUN make install
 
 FROM debian:bullseye-slim
 
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends ca-certificates \
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/* \
+  && update-ca-certificates
```

Comment:
I think `update-ca-certificates` doesn't need to be run because there is no self-signed certificates.
https://manpages.ubuntu.com/manpages/xenial/man8/update-ca-certificates.8.html

Reply:
Ahh, you right. Fixed in https://github.com/crowdworks/slacts/pull/41/commits/a9f5b4f0c6b22d0b0d0b4536c5166165cfd6486d